### DISCUSSION TITLE
Add PackedDictionaryArray

### DISCUSSION THREAD

**Discussion by InkONat** [godotengine/godot-proposals#8690] [Engine Core]
Basically a packed Array of dictionaries, I don't know what else to say here. This could be useful to save information on an arbitrary number of objects as dictionaries.

**Comment by AThousandShips** [comment]
This has been discussed before and been determined not to be needed, the packed arrays are intended for high performance areas transferring data from the engine to scripts, and this simply isn't needed anywhere, and `Dictionary` is a slow type so adding a more performant type for it doesn't make sense

You can still store things in an `Array[Dictionary]`, no need for a packed type, the only reason to have a PackedArray type is to handle performance sensitive areas, the main area for this is the renderer

**Comment by Mickeon** [comment]
What would be the benefit over `Array[Dictionary]`?
